src/components/CentralHeader/CentralHeader.tsx

```diff
@@ -31,22 +32,42 @@ interface ICentralHeaderProps extends RouteComponentProps {
 /**
  * Central header right below Slim header, containing app title and user name and user role with dropdown to go to profile page and logout
  */
-export const CentralHeader = withRouter<
-  ICentralHeaderProps,
-  React.FC<ICentralHeaderProps>
->((props: ICentralHeaderProps) => {
+export const CentralHeader = withRouter((props: ICentralHeaderProps) => {
   /**
    * react-i18next translation hook
    */
   const { t } = useTranslation();
 
+  /**
+   * add user icon if user name is set
+   */
+  const userIcon =
+    props.userName !== "" && props.userName !== " " ? (
```

Comment:
why `userName` could be "" or " " ?

Reply:
as union of two strings from the getProfile service (family_name and given_name), I had to join these strings with a whitespace and check for the " "; now i substituted the check with string.trim() to join both checks